src/pages/questionPage.js

```diff
@@ -28,13 +30,32 @@ export const initQuestionPage = () => {
   document
     .getElementById(NEXT_QUESTION_BUTTON_ID)
     .addEventListener('click', nextQuestion);
+
+    let timerElement = createTimerElement(second);
+    questionElement.appendChild(timerElement);
+  
+    const timerInterval = setInterval(()=>{
+      if(second !== 1){
+        second -= 1;
+    questionElement.removeChild(timerElement);
```

Comment:
are you sure you are running the command `npm run prettier:write`? I can see tabs are not correct here.

Reply:
let me do it.